tuf/store/filestore.go

```diff
@@ -53,7 +55,14 @@ func (f *FilesystemStore) GetMeta(name string, size int64) ([]byte, error) {
 		}
 		return nil, err
 	}
-	return meta, nil
+	if size == -1 {
```

Comment:
@riyazdf good point on `strings.Replace`. We should follow what Go's core libs are already doing. I'll update listing timestamps to use `-1`